internal/controller/api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -103,6 +103,11 @@ type DebugInfo struct {
 	// LogsURL specifies a url for download controller logs tar file.
 	// +optional
 	LogsURL string `json:"logsURL"`
+	// Current state of the AgentClusterInstall
+	State string `json:"state,omitempty"`
```

Comment:
Same like written in Agent regarding the place of the fields (should be under `AgentClusterInstallStatus`) and the naming of the fields.